operations/experimental/example-vars-files/vars-override-otel-collector-exporters.yml

```diff
@@ -1,6 +1,6 @@
 ---
 otel_collector_metric_exporters:
   file/test:
-    path: /var/vcap/sys/log/otel-collector/file.log
+    path: /tmp/otel-collector-file.log
```

Comment:
I know that this is an experimental release and test config, so maybe therefore you have changed the path to the log file.
I don't think that this is good because of the following:
- All BOSH deployments write their logs in `/var/vcap/log/<job-name>`. If the path is changed it will be confusing for the users
- If the logs is written in `/tmp` the logs will not be persisted in-between restarts of the VM as the `/tmp`. This would make troubleshooting harder
- The filename should also follow the BOSH conventions `<jobname>.stdout.log`, `<jobname>.stderr.log` and it would be good if the custom logs have meaningful names as well

Reply:
We made this change so that in CI the log files for both otel-collector and otel-collector-windows can go to the same place, so we can deploy them with the same variable. On windows the /var/vcap/sys/log/otel-collector/ path doesn't exist. In general we don't expect users to use the file exporter except for testing.

Some Options:
1. We could use a different vars file in CI, however that has the downside of leaving this vars file untested in the logging and metrics pipelines
1. We could use two variables, but this has the downside of not sharing the exporter config across all the collectors
1. Leave it as is, and expect users to not copy this vars file verbatim

What do you think?

Reply:
Thank you very much for the detailed  explanation @mkocher. I guess it's ok if we move the log file to `/tmp`. The component is in early stage and still experimental. We would have to think about how to configure it properly before going GA.